examples/website/geojson/app.js

```diff
@@ -30,14 +36,18 @@ export const COLOR_SCALE = scaleThreshold()
     [128, 0, 38]
   ]);
 
-const LIGHT_SETTINGS = {
-  lightsPosition: [-125, 50.5, 5000, -122.8, 48.5, 8000],
-  ambientRatio: 0.2,
-  diffuseRatio: 0.5,
-  specularRatio: 0.3,
-  lightsStrength: [2.0, 0.0, 1.0, 0.0],
-  numberOfLights: 2
-};
+const ambientLight = new AmbientLight({color: [255, 255, 255], intensity: 1.0});
+const pointLight = new DirectionalLight({
+  color: [255, 255, 255],
+  intensity: 2.0,
+  direction: [5446, 6114, -200]
```

Comment:
Perhaps we want to discuss what type of direction parameters to use - is this a vector? Can we use degrees etc (perhaps via math.gl SphericalCoordinates class).